### DISCUSSION TITLE
SnowCLI 2.0 Documentation & Examples

### DISCUSSION THREAD

**Discussion by kokorin** [Snowflake-Labs/snowcli#777] [Q&A] [answered]
It looks like [the guide](https://docs.snowflake.com/LIMITEDACCESS/snowcli/snowcli-guide) references 1.x versions and not the latest 2.0 release.

**Comment by fletchertyler914** [comment]
would love v2 docs as well.

**Comment by sfc-gh-anugupta** [comment, marked as answer]
https://docs.snowflake.com/en/developer-guide/snowflake-cli-v2/index
